docs/book/stacks-and-components/component-guide/orchestrators/skypilot-vm.md

```diff
@@ -374,6 +414,34 @@ skypilot_settings = SkypilotAzureOrchestratorSettings(
 )
 ```
 
+{% endtab %}
+
+{% tab title="Lambda" %}
+
+**Code Example:**
+
+```python
+from zenml.integrations.skypilot_lambda.flavors.skypilot_orchestrator_lambda_vm_flavor import SkypilotLambdaOrchestratorSettings
```

Comment:
```suggestion
from zenml.integrations.skypilot_lambda import SkypilotLambdaOrchestratorSettings
```

Here the import could be shorter right? I think it is a bit more readable that way.